### PR TITLE
Upgrade to Polars 0.22.8

### PR DESCRIPTION
**Pull request by joshuataylor**
This allows us the next step of upgrading to support Arrow IPC Streams.

Let me know what the thoughts around window_size_duration is, I think that's right?

### CODE REVIEW

File: native/explorer/src/series.rs [hunk 1]
```diff
@@ -308,8 +308,8 @@ pub fn s_len(data: ExSeries) -> Result<usize, ExplorerError> {
 #[rustler::nif(schedule = "DirtyCpu")]
 pub fn s_fill_none(data: ExSeries, strategy: &str) -> Result<ExSeries, ExplorerError> {
     let strat = match strategy {
-        "backward" => FillNullStrategy::Backward,
-        "forward" => FillNullStrategy::Forward,
+        "backward" => FillNullStrategy::Backward(None),
+        "forward" => FillNullStrategy::Forward(None),
```
Comment (cigrainger) [inline native/explorer/src/series.rs:312]: Do you know what the enum value is for here? I couldn't find it in the docs.

**Review by cigrainger — APPROVED** [review]
I think the window size duration looks right to me. And the tests pass :).

**Review by philss — APPROVED** [review]
LGTM! 

For what I could understand, the change related to the `window_size` is correct. :+1:

File: native/explorer/src/series.rs (hunk 1 above)
Comment (joshuataylor) [reply native/explorer/src/series.rs:312]: So, had a dig as I was also curious!

Boils down to this issue: https://github.com/pola-rs/polars/issues/3545
Which was implemented here: https://github.com/pola-rs/polars/pull/3559

It's defined in

 https://docs.rs/polars/latest/polars/prelude/enum.FillNullStrategy.html

Comment (cigrainger) [reply native/explorer/src/series.rs:312]: Ohhhh interesting! Okay! Thanks for finding that :).

**Pull request merged**